client-lib/src/main/java/com/ibm/ws/repository/strategies/writeable/AddThenHideOldStrategy.java

```diff
@@ -174,7 +119,4 @@ private boolean isVisible(RepositoryResourceImpl resource) {
         return displayPolicy == DisplayPolicy.VISIBLE || displayPolicy == null;
```

Comment:
Are you sure we should be returning `true` for non-`WebDisplayable` resources? That seems iffy.